packages/versioning/lib/decorators.cadl

```diff
@@ -14,6 +14,7 @@ extern dec madeOptional(target: unknown, version: EnumMember);
 
 extern fn addedAfter(target: unknown, version: EnumMember): boolean;
 extern fn removedOnOrBefore(target: unknown, version: EnumMember): boolean;
+extern fn hasDifferentNameAtVersion(target: unknown, version: EnumMember): boolean;
 extern fn renamedAfter(target: unknown, version: EnumMember): boolean;
```

Comment:
would maybe remove renamedAfter from there, no need to get signature help for it.

Reply:
Removed the deprecated signatures.